week_1/css_advanced_and_best_practices/portfolio-styles.css

```diff
@@ -91,7 +87,8 @@ form button {
   border-radius: 5px;
   cursor: pointer;
 }
-
 form button:hover {
   background-color: #c0392b;
 }
+
+/*# sourceMappingURL=portfolio-styles.css.map */
```

Comment:
**Suggestion:** The source mapping URL comment at the end of the file is unnecessary and can be removed.
```suggestion

```